### PR TITLE
feat: Expiration and time column of offers on profile

### LINKED ISSUE #11028: Expiration and time column of offers on profile
image

so what does it say? 
image

17.9. was already, so is it just expired offer? 

image

please match it to the figma design please (changing time to "expiration") 

when its expired, change the "date info" to "expired" and put a "withdraw funds" in medium outlined button instead of cancel offer

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #11028

## Needs Design check

- @exezbcz please review

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

CleanShot 2024-09-23 at 13 27 11@2x

### CODE REVIEW

**Comment by vikiival** [comment]
Btw, speck v14 has fixed expiration :)

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
> (No funds are reserved when creating an offer)

that's true 

So let's call it somehow else? e.g `Withdraw offer`
cc @exezbcz

**Comment by exezbcz** [comment]
@vikiival so why we are doing auto offer refund after they are expired? 
 

is not that because of the funds are reserved?

File: components/profile/activityTab/OfferRow.vue [hunk 1]
```diff
@@ -139,10 +144,15 @@
       <div v-else>
         {{ blank }}
       </div>
-      <div v-if="offer.expirationDate">
-        <span> {{ format(offer.expirationDate, EXPIRATION_FORMAT) }}</span>
-        <span class="text-k-grey ml-3"> ({{ formatToNow(offer.expirationDate, isExpired) }})</span>
-      </div>
+      <template v-if="offer.expirationDate">
+        <div v-if="ieExpired">
```
Comment (vikiival) [inline components/profile/activityTab/OfferRow.vue:148]: typo here?

Comment (hassnian) [reply components/profile/activityTab/OfferRow.vue]: nice catch, don't know how this one slipped through

**Pull request merged**